draft-ietf-core-href.md

```diff
@@ -583,19 +617,28 @@ references:
 * An empty path in a `CRI` MUST be represented as the empty array
```

Comment:
This has become a factual statement now (when previously there was some choice that required force):

```suggestion
* An empty path in a `CRI` is be represented as the empty array
```

(Not a blocking comment: it's not wrong, just IMO unnecessary)